packages/main/src/components/top-navigation-bar/TopNavigationBar.jsx

```diff
@@ -18,6 +18,10 @@ import ar from "./assets/language/ar.png";
 import he from "./assets/language/he.png";
 import pt from "./assets/language/pt.png";
 import world from "./assets/language/world.png";
+import pp from "./assets/language/pp.png";
```

Comment:
Give this image a more descriptive name. `pp` could mean anything

Reply:
ive resolved this